src/app/TargetView/TargetView.tsx

```diff
@@ -45,6 +46,7 @@ import { NoTargetSelected } from './NoTargetSelected';
 import { TargetContextSelector } from './TargetContextSelector';
 
 interface TargetViewProps {
+  appendDashboard?: boolean;
```

Comment:
I think this could be a prop like: 

```ts
attachments?: React.ReactNode;
```

This way, parent can supply any attachments needed (easy to extend later if other views want to add something below context selector). Wdu think?